packages/nc-gui/composables/useMultiSelect/convertCellData.ts

```diff
@@ -0,0 +1,38 @@
+import { UITypes } from 'nocodb-sdk'
+
+export default function convertCellData(args: { from: UITypes; to: UITypes; value: any }) {
```

Comment:
I think it'd better to not allow invalid data being pasted or we can show the message saying the data type doesn't match with the column etc. E.g. pasting a string to a numeric field.

Reply:
If the parsed data is not valid we can throw an error, will take care of that part :+1: